docs/install-development-cluster.md

```diff
@@ -0,0 +1,25 @@
+# Installing in a Development Environment
+
+> [!NOTE]
+> This instalation is meant for testing purposes only. It uses self signed certificates, so your browser will show a warning about it.
+
+To install Tracetest OnPrem in a development environment, and get a first glimpse of how it works, you can follow the instructions below.
+
+We provide a development install via a script that you can use to run Tracetest OnPrem locally on your machine. The only prerequisite is that you have [kind](https://kind.sigs.k8s.io/) installed.
+
+You can then run the following command and follow the instructions when needed:
+```sh
+curl -sSL https://raw.githubusercontent.com/kubeshop/tracetest-cloud-charts/main/scripts/setup_kind_cluster.sh | bash -- --install-install-demo
```

Comment:
```suggestion
curl -sSL https://raw.githubusercontent.com/kubeshop/tracetest-cloud-charts/main/scripts/setup_kind_cluster.sh | bash -- --install-demo
```

Reply:
I'm not sure this will work like this. the script assumes it's a local copy of the repo, and doesn't add this tracetest repo. instead it references the files with paths relative to the scriptl

Reply:
Oh, I'll take a look at that, thanks!